tools/adventure-pack/goodies/typescript/Iterator.prototype.forEach/index.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @goody {}
+ */
+
+import "../Iterator.prototype.toIterable";
+import { iteratorPrototype } from "../Iterator.prototype";
+
+declare global {
+  interface Iterator<T> {
+    forEach(callbackFn: (element: T, index: number) => void): void;
```

Comment:
Does `=> void` in the callback signature mean that we get an error if we use a callback that returns something? Could we add that as a test?

It's slightly bad style to return something from the `.forEach` callback in my opinion, but we shouldn't forbid it.